examples/with-thirdpartyemailpassword-express/index.js

```diff
@@ -0,0 +1,133 @@
+
+supertokens.init({
+    framework: "express",
+    supertokens: {
+        // TODO: This is a core hosted for demo purposes. You can use this, but make sure to change it to your core instance URI eventually.
+        connectionURI: "https://try.supertokens.io",
+        apiKey: "<REQUIRED FOR MANAGED SERVICE, ELSE YOU CAN REMOVE THIS FIELD>",
+    },
+    appInfo: {
+        appName: "SuperTokens Demo App", // TODO: Your app name
+        apiDomain, // TODO: Change to your app's API domain
+        websiteDomain, // TODO: Change to your app's website domain
+    },
+    recipeList: [
+        ThirdPartyEmailPassword.init({
+            /*
+                We use different credentials for different platforms when required. For example the redirect URI for Github
+                is different for Web and mobile. In such a case we can provide multiple providers with different client Ids.
+                
+                When the frontend makes a request and wants to use a specific clientId, it needs to send the clientId to use in the 
+                request. In the absence of a clientId in the request the SDK uses the default provider, indicated by `isDefault: true`.
+
+                When adding multiple providers for the same type (Google, Github etc), make sure to set `isDefault: true`.
+            */
+            providers: [
+                // We have provided you with development keys which you can use for testing or when running our demo apps.
+                // IMPORTANT: Please replace them with your own OAuth keys for production use.
+                ThirdPartyEmailPassword.Google({
+                    isDefault: true,
+                    clientId: "1060725074195-kmeum4crr01uirfl2op9kd5acmi9jutn.apps.googleusercontent.com",
+                    clientSecret: "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",
+                }),
+                ThirdPartyEmailPassword.Google({
+                    clientId: "1060725074195-c7mgk8p0h27c4428prfuo3lg7ould5o7.apps.googleusercontent.com",
+                    clientSecret: "",
+                }),
+                ThirdPartyEmailPassword.Github({
+                    isDefault: true,
+                    clientSecret: "e97051221f4b6426e8fe8d51486396703012f5bd",
+                    clientId: "467101b197249757c71f",
+                }),
+                ThirdPartyEmailPassword.Github({
+                    clientSecret: "00e841f10f288363cd3786b1b1f538f05cfdbda2",
+                    clientId: "8a9152860ce869b64c44",
+                }),
+                /*
+                    For Apple signin, iOS apps always use the bundle identifier as the client ID when communicating with Apple. Android, Web and other platforms
+                    need to configure a Service ID on the Apple developer dashboard and use that as client ID.
+
+                    In the example below 4398792-io.supertokens.example.service is the client ID for Web. Android etc and thus we mark it as default. For iOS
```

Comment:
Should be a comma instead of a dot in "Web. Android"

Reply:
Remove "and thus we mark it as default"

Reply:
Done